package.json

```diff
@@ -46,6 +46,7 @@
     "apollo-server-koa": "^2.0.0",
     "ava": "^0.25.0",
     "aws-sdk-mock": "^4.0.0",
+    "aws-xray-sdk-core": "^1.3.0 || ^2.0.0",
```

Comment:
This semver pattern is strange to me and I have seen it behave in ways I didn't expect when consumed by other projects. Could you help me understand why including two different versions is useful?

Reply:
I know that our existing projects have a mix of `aws-xray-sdk-core` versions and I didn't want this upgrade for force a change that wasn't needed.

Reply:
What I have seen in practice is that it can still force a change but is harder to understand. I saw one case where tests were assuming that the test instance of xray and the instance in the app code where the same instance. However, the app code picked up version `1.3.0` and the test code got `2.0.0` as `yarn` attempted to use the highest version possible. Granted, this was ultimately an instance of poor dependency management, I still think it would have been easier to reason about if the dependency statement had specified exactly one version.

An alternative approach that I have seen to managing the upgrade problem is to update the major version number when a significant dependency has a major version upgrade. Thoughts?

Reply:
For simplicity, I'm fine with upgrading to 2.x and bumping the version of lambda-tools. I'll do that now

Reply:
The version is currently 5.x-beta already so I didn't change it as part of this PR